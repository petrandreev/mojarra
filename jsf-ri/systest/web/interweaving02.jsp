<!--
Copyright 2004 Sun Microsystems, Inc. All rights reserved.
SUN PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
-->
<%@ page contentType="text/html" language="java" %>
<%@ taglib prefix="c" uri="http://java.sun.com/jstl/core" %>
<%@ taglib prefix="f" uri="http://java.sun.com/jsf/core" %>
<%@ taglib prefix="h" uri="http://java.sun.com/jsf/html" %>

<f:view>
    <html>
    <head>
        <title>interweaving01</title>
    </head>

    <body>
    <h:outputText value="Begin"/>
    test
    <h:outputText value="jstl import without verbatim"/>

    <c:import url="test01.jsp"/>

    <h:outputText value="End"/>
    test
    <h:outputText value="jstl import without verbatim"/>
    </body>
    </html>
</f:view>


