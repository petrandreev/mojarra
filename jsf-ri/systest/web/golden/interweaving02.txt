<!--
 Copyright 2004 Sun Microsystems, Inc. All rights reserved.
 SUN PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
-->






<html>
<head>
<title>interweaving01</title>
</head>
<body>
Begin
test
jstl import without verbatim
interweaving
<!--
 Copyright 2004 Sun Microsystems, Inc. All rights reserved.
 SUN PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
-->





works
well!!
End




test
jstl import without verbatim
</body>
</html>



